server/apps/distributed-app/docs/modules/ROOT/pages/helm/checklist.adoc

```diff
@@ -0,0 +1,43 @@
+# Checklist
+
+There is some things that you must have before being able to run James on your Kubernetes Cluster.
+
+## MUST have
+
+Those are absolutely necessary if you want at least James to start without crashing.
+
+- [ ] Setup RabbitMQ and create James user in it.
+- [ ] Create a Kubernetes namespace for James deployment.
+- [ ] A `distributed` version of James image.
+- [ ] Have an appropriate email domain (Contact your Domain Service Provider)
+- [ ] Cassandra nodes deployed.
+- [ ] An ObjectStorage compatible with S3 APIs.
+- [ ] Configure JWT keys, keystores, SPF and DKIM. See document [here](https://james.apache.org/howTo/)
+- [ ] Adding the proper `mailetcontainer.xml` file with other conf files in James Helm package
```

Comment:
I'm not sure I agree on this one. We did like this because we were trying to have our conf files centralized and for `mailetcontainer.xml` it was just not possible with different tenants.

I think we should provide a default mail processing pipeline here for the community doc, as it's easy to forget. It's up to the people using it after to reorganize it as they want (you could think as well other conf files than mailetcontainer could be hard to be centralized depending on multiple setups for other people, but our doc should not solve that)

Reply:
There is one sample `mailetcontainer.xml` in [here ](https://github.com/apache/james-project/blob/master/server/apps/distributed-app/sample-configuration/mailetcontainer.xml).

I think i should check through all the docs again and put the link to that files whenever possible.

Reply:
the `mailetcontainer.xml` explain + link to sample is alr in `run/run-kubernetes.adoc`
```
=== James Configuration

The configuration files proper to James are stored in the Chart `james/configs` folder. They are being stored in a `ConfigMap` and then being mounted in the James pod at /root/conf path.

If you need to change a conf in James, this is were you need to look at.

For now all configuration files are the same for every deployment, except for `mailetcontainer.xml` that has been excluded (see section below).

You can get the template `mailetcontainer.xml` in our link:https://github.com/apache/james-project/blob/master/server/apps/distributed-app/sample-configuration/[sample-configuration] folder.
```